nixos/modules/programs/sway.nix

```diff
@@ -56,7 +66,7 @@ in
     environment.systemPackages = [ swayJoined ] ++ cfg.extraPackages;
     security.wrappers.sway = {
       program = "sway-setcap";
-      source = "${sway}/bin/sway";
+      source = "${swayPackage}/bin/sway";
       capabilities = "cap_sys_ptrace,cap_sys_tty_config=eip";
       owner = "root";
       group = "sway";
```

Comment:
`permissions = "u+rx,g+rx";`

@Mic92 BTW is there any specific reason to require the `sway` group here? Normally I would expect `user=root`, `group=root` and `ugo=rx` so that every user could execute Sway. Are there any security concerns or did I miss something else?

Reply:
This would grant every user to get the `ptrace` capability and therefor potentially control every other process regardless of uid. Therefore it should be handled with care.